packages/electron-app/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "@polkadot/electron-app",
```

Comment:
Would probably name this `apps-electron` (Just organizes the `@polkadot/*` namespace a slightly better)

Reply:
done